README.md

```diff
@@ -52,13 +52,29 @@ The i8042 PS/2 controller emulates, at this point, only the
 [CPU reset command](https://wiki.osdev.org/%228042%22_PS/2_Controller#CPU_Reset)
 which is needed for announcing the VMM about the guest's shutdown.
 
+## ARM PL031 Real Time Clock
+
+This module emulates the ARM PrimeCell Real Time Clock (RTC)
+[PL031](https://developer.arm.com/documentation/ddi0224/c/Functional-overview/RTC-operation/RTC-operation).
+The PL031 provides a long time base counter with a 1HZ counter signal and
+a configurable offset.
+
+This implementation emulates all control, peripheral ID, and PrimeCell ID
+registers; however, the interrupt based on the value of the Match Register
+(RTCMR) is not currently implemented (i.e., setting the Match Register has
+no effect).
+
+For a VMM to be able to use this device, the VMM needs to do the following:
+- add the RTC to the Bus (either PIO or MMIO)
+- event handling (optional)
```

Comment:
Agreed.  I've removed this and added a note so that it's explicit that no event handling is required.  The note also serves as an implicit reminder that if we ever do implement the Match Register, the README description needs to be updated.